Rakefile

```
@@ -21,7 +21,9 @@ require_relative 'lib/rubocop/cop/generator'
 
 desc 'Run tests'
 task :test do
-  system("bundle exec minitest-queue #{Dir.glob('test/**/*_test.rb').shelljoin}")
+  error_on_failure = ENV.fetch('ERROR_ON_TEST_FAILURE', 'true') != 'false'
+
+  system("bundle exec minitest-queue #{Dir.glob('test/**/*_test.rb').shelljoin}", exception: error_on_failure)
```

Is there any problem with always setting it to `true` for `exception` option?

```suggestion
  system("bundle exec minitest-queue #{Dir.glob('test/**/*_test.rb').shelljoin}", exception: true)
```

---

Yes, two of the jobs are currently failing and I think fixing them should be done in dedicated PRs - once they're fixed, this can then be set to `true`.

---

Ah, OK. Let's merge this first 👍